gradle/README.md

```diff
@@ -0,0 +1,6 @@
+# Gradle
+
+To regenerate the [project-schema.json](project-schema.json) file run this in the project's root directory:
+```bash
+./gradlew kotlinDslAccessorsSnapshot
```

Comment:
You want me to put something in about it?

From what I know it's simply gradle way of figuring out what all of the extensions are after all of the plugins have settled and fully defined all of their configurations.